src/rat/plugins/forecasting.py

```diff
@@ -419,8 +911,21 @@ def forecast(config, rat_logger):
     raw_gfs_dir = gfs_dir / 'raw'
     extracted_gfs_dir = gfs_dir / 'extracted'
     processed_gfs_dir = gfs_dir / 'processed'
-
-    for d in [raw_gefs_chirps_dir, processed_gefs_chirps_dir, raw_gfs_dir, extracted_gfs_dir, processed_gfs_dir]:
+    inflow_dst_dir = basin_data_dir / 'rat_outputs' / 'forecast_inflow' / f"{basedate:%Y%m%d}"
+    basin_reservoir_shpfile_path = Path(basin_data_dir) / 'gee' / 'gee_basin_params' / 'basin_reservoirs.shp'
+    final_inflow_out_dir = basin_data_dir / 'final_outputs' / 'forecast_inflow' / f"{basedate:%Y%m%d}"
+    final_evap_out_dir = basin_data_dir / 'final_outputs' / 'forecast_evaporation' / f"{basedate:%Y%m%d}"
+    evap_dir = basin_data_dir / 'rat_outputs' / 'forecast_evaporation' / f"{basedate:%Y%m%d}"
+    rule_curve_dir = Path('/cheetah2/pdas47/rat3_mekong/global_data/rc')    # TODO: change to dynamic
```

Comment:
This path needs to be changed to a generic one.

Reply:
Changed the code to read the path from the config file. The added option is under `['PLUGINS']['forecast_rule_curve_dir']`.